subsys/bluetooth/host/audio/mpl.c

```diff
@@ -981,6 +967,31 @@ static bool do_next_track(struct mpl_mediaplayer_t *pl)
 	return track_changed;
 }
 
+/* Next track, when the next track has been explicitly set
+ *
+ * ALWAYS changes the track, changes the group if required
+ * Resets the next_track_set and the "next" pointers
+ *
+ * Returns true if the _group_ has been changed, otherwise false
+ */
+static bool do_next_track_set(struct mpl_mediaplayer_t *pl)
```

Comment:
```suggestion
static bool next_track_set(struct mpl_mediaplayer_t *pl)
```
or
```suggestion
static bool set_next_track(struct mpl_mediaplayer_t *pl)
```

Reply:
Hmm, I'm getting confused about the naming. What's the difference between this function and `do_next_track`?

Reply:
Renamed the functions to do_next_track_normal_order() do_next_track_next_track_set().
The first one changes according to the preset order (goes to the current track's next track), the second one changes to the track that has been set as the next track.

Reply:
Can you briefly explain why we need two different functions for changing to the next track?

Reply:
Well, they can of course be hidden under a common wrapper (or merged to one function).
But having them as two is clearer, as one of them is a special case, while the other follows the pattern of all the other "do_<prev|nest|first|last|goto>_<segment|track|group>.

Reply:
Also, they behave slightly differently with respect to the track position.